src/github3/repos/repo.py

```diff
@@ -2435,8 +2435,9 @@ def subscribe(self):
             :class:`~github3.notifications.RepositorySubscription`
         """
         url = self._build_url("subscription", base_url=self._api)
+        new_subscription = {"subcribed": subscribed, "ignored": ignored}
```

Comment:
Thanks for your contribution! It seems like we had a typo here, and the key name was misspelled as "subcribed". Can you fix this too, while you're here?